ionoscloud/resource_server.go

```diff
@@ -1173,9 +1162,12 @@ func resourceServerImport(ctx context.Context, d *schema.ResourceData, meta inte
 	}
 
 	if len(parts) > 3 {
```

Comment:
@cristiGuranIonos can you add some info about how the ID would look like with more than 3 parts? Above we check if `len(parts) < 2` and you can see why since we have this:
```
Expecting {datacenter UUID}/{server UUID}"
```
So we kinda know what to expect. Here in the case of `len(parts) > 3` is unclear, I suppose it is something like `dcUUID/serverUUID/nicUUID/firewallRuleUUID` but I'm not sure, please add a comment related to this.

Reply:
yes, that is how we support the import. will add a comment

Reply:
@cristiGuranIonos the comment it's not added yet.